hack/gen_tilt_settings.sh

```diff
@@ -25,7 +25,14 @@ function get_latest_release() {
     release="$(curl -H "Authorization: token ${GITHUB_TOKEN}" -sL "${1}")" || ( set -x && exit 1 )
   fi
   # This gets the latest release as vx.y.z , ignoring any version with a suffix starting with - , for example -rc0
+  # Uncomment the following line and delete the next line when we have a proper CAPI 1.1.x release. Using pre-release for time being
+  # release_tags_unsorted="$(echo "$release" | jq -r "[.[].tag_name | select( startswith(\"${2:-""}\")) | select(contains(\"-\")==false)]" \
   release_tag="$(echo "$release" | jq -r "[.[].tag_name | select( startswith(\"${2:-""}\")) | select(contains(\"-\")==false)] | max ")"
```

Comment:
this looks not used anywhere?

Reply:
I think I have messed up the commit. That doesn't look like the commit I tested

Reply:
Let me check